src/Compute/Compute/help/New-AzDiskConfig.md

```diff
@@ -432,6 +433,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -PurchasePlan
+{{ Fill PurchasePlan Description }}
```

Comment:
If no help message given, this part will be empty.

Reply:
Added